lib/WebServer/LapTriggerWebServer.cpp

```diff
@@ -250,6 +227,106 @@ void LapTriggerWebServer::handleCredentials()
     }
 }
 
+void LapTriggerWebServer::parseWSTextEvent(uint8_t clientId, WStype_t type, uint8_t *payload, size_t length)
```

Comment:
Is payload a output parameter?

Reply:
No, it is an input array. I added const to these parameters